packages/kbn-test/src/functional_tests/lib/config_path.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import Path from 'path';
+
+import { Flags } from '@kbn/dev-cli-runner';
+import { createFlagError } from '@kbn/dev-cli-errors';
+
+const makeAbsolute = (path: string) => Path.resolve(path);
+
+function iter(flags: Flags, key: string) {
```

Comment:
This function and [`packages/kbn-test/src/functional_test_runner/cli.ts:33` ](https://github.com/elastic/kibana/pull/140680/files#diff-1fd17763545c1c7045b33191f8bdb702eb66e48762576c745e18484dcf4f549cR33) looks quite similar maybe we can reuse